src/helics/apps/Clone.cpp

```diff
@@ -374,9 +379,11 @@ std::shared_ptr<helicsCLI11App> Clone::buildArgParserApp()
     app->add_flag("--allow_iteration", allow_iteration, "allow iteration on values")
         ->ignore_underscore();
 
-    app->add_option("--output,-o", outFileName, "the output file for recording the data")
-        ->capture_default_str();
-    app->add_option("capture", captureFederate, "name of the federate to clone");
+    app->add_option(
+        "--capture,capture",
```

Comment:
What does the comma-separated duplicate of it do? Allow it to be a list?

Reply:
the `--capture` defines an option,  `capture` defines a positional,   The name really isn't that important for functionality, it just indicates this can be a positional option.